lua/apisix/plugins/proxy-rewrite.lua

```diff
@@ -82,6 +82,16 @@ function _M.check_schema(conf)
         return false, err
     end
 
+    if conf.regex_uri and #conf.regex_uri > 0 then
+        local _, _, err = re_sub("/fake_uri", conf.regex_uri[1],
```

Comment:
what is `/fake_uri`? why we need it?

Reply:
The `/fake_uri` is used to verify the validity of regex expression.

Reply:
need a test case to cover invalid regex.

Reply:
https://github.com/apache/incubator-apisix/pull/979/files#diff-0dd1fb219e34e914ab6423384290b98cR895

Is this test case?

Reply:
not this one, this is for invalid `$1`